src/dxitemgroup/dropdown.scss

```diff
@@ -26,7 +26,7 @@
 		position: absolute;
 		bottom: 12px;
 		right: 12px;
-		content: '\f105';
+		content: '⌄';
```

Comment:
I don't think this css class is getting used anywhere but I replaced with a down caret just in case.